src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -749,6 +860,15 @@ components:
         - dispatched
         - not-delivered
 
+    SearchBy:
```

Comment:
This might be a bit too vague/generic (e.g. if another search endpoint is added in the future which uses different search fields), but I'm struggling to think of a better name! 

 `AedSearchByField`? 🤷‍♂️